TheWarWithin/RogueAssassination.lua

```diff
@@ -2644,18 +2492,17 @@ spec:RegisterAbilities( {
         spendType = "energy",
 
         startsCombat = true,
-
+        usable = function() return covenant.night_fae and true or false end,
```

Comment:
`function() return covenant.night_fae end` is suffcient

Reply:
ok!